privacy-snapshot.json

```diff
@@ -48,6 +48,8 @@
   "price.api.cx.metamask.io",
   "proxy.api.cx.metamask.io",
   "proxy.dev-api.cx.metamask.io",
+  "qtgdj2huxh.execute-api.us-east-2.amazonaws.com",
+  "https://signature-insights.api.cx.metamask.io",
```

Comment:
Where is this entry coming from?

Reply:
These are UAT and PROD url for signature decoding api.

Reply:
I removed PROD url for now, we can add it back once we switch to prod instance